test/test.pl

```diff
@@ -23,4 +23,31 @@
     string_codes(Actual, Codes),
     Expected = "Goodbye, Tokenize!".
 
+
+test('tokenize 7.0',
```

Comment:
Should we test scientific notation and negatives etc? Or is does it not yet support that?

Reply:
supports anything SWI-Prolog supports, as the actual parsing uses dcg/basics number//1. 
In turn that uses the Prolog parser itself, so testing number formats didn't seem that interesting.
I can add some tests for those cases if you want.

Reply:
Oh no that makes sense.